src/Microsoft.CSharp/src/Microsoft.CSharp.csproj

```diff
@@ -8,6 +8,7 @@
     <!-- dotnet/corefx#3128 tracks removing this exclusion -->
     <ExcludeLocalizationImport Condition="'$(TargetsWindows)' != 'true'">true</ExcludeLocalizationImport>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.3;netcore50</PackageTargetFramework>
```

Comment:
Won't conflict.  Explicit assets always win.  That's why this commit works even without the buildtools change.